packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.spec.js

```diff
@@ -465,7 +467,8 @@ describe('Component > VisXZoom', function () {
           clientX: 50,
           clientY: 50,
           deltaY: -1,
-          preventDefault: sinon.spy()
+          preventDefault: sinon.spy(),
+          [ZOOM_HOT_KEY]: true
```

Comment:
I forgot that the tests don't need the modifier key unless you also pass `disablesScrolling={false}` into the `VisXZoom` component. Default behaviour is to continue as before, to avoid breaking the JSON data viewers.